packages/core/addon/templates/components/navi-visualization-config/line-chart.hbs

```diff
@@ -1,12 +1,11 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#component
-  "navi-visualization-config/series-chart"
-  request=request
-  response=response
-  seriesConfig=seriesConfig
-  seriesType=seriesType
-  onUpdateConfig=(action "onUpdateConfig")
-}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<NaviVisualizationConfig::SeriesChart
+  @request={{request}}
```

Comment:
should these be `@request={{@request}}`? I know it's in core so it's not on octane yet, but it should still work I think

Reply:
Yea it should work, I was making minimal changes in other packages to satisfy template lint